docs/reference/core/load-balancer.md

```diff
@@ -42,10 +42,13 @@ Configuring sticky sessions makes Ambassador route requests to the same backend
 - Header
 - Source IP
 
+**NOTE:** Ambassador supports sticky sessions using 2 load balancing policies, `ring_hash` and `maglev`.
+
+
 ##### Cookie
 ```yaml
 load_balancer:
-  policy: ring_hash
+  policy: ring_hash/maglev
```

Comment:
This will probably have some folks thinking that the literal string `ring_hash/maglev` is what you should type. Maybe just pick one, and note separately that the other is supported too?

Reply:
Makes sense, fixed :)